Todo.py

```diff
@@ -0,0 +1,203 @@
+            if len(message.embeds) > 0: # Check if it's an embed, I think this will avoid most problems
+                if reaction.user_id != bot.user.id:
+                    if reaction.emoji.name == EMOJIS['check']:
+                        await message.add_reaction(EMOJIS['hourglass'])
+                    elif reaction.emoji.name == EMOJIS['hourglass']:
+                        await message.add_reaction(EMOJIS['check'])
+
+    @commands.group()
+    async def todo(self, ctx):
+        '''Commands to manage a todolist.'''
+        if ctx.invoked_subcommand is None:
+            await ctx.send('Error: See for ``' + PREFIX + 'help todo``')
+
+    @todo.command()
+    async def add(self, ctx, *args):
+        '''Command to add a todo. Usage : <the thing todo>;<type of todo>;<assigned to / false>;<repost public channel / false>'''
+        with open(TODO_CHANNEL, "r") as file:
+            channelId = file.readlines()[0]
+            if channelId == None or channelId == "":
+                await ctx.channel.send("The todos' channel must be selected with the command " + PREFIX + "todo channel")
+                return
+        channel = bot.get_channel(int(channelId))
+
+        command = ""
+        for arg in args:
+            command += " {}".format(arg)
+        command = command.split(";")
+
+        todoType = None
+
+        with open(TODO_TYPES, "r") as file:
+            lines = file.readlines()
+            for line in lines:
+                line = line.split(';')
+                if command[1] == line[0]:
+                    todoType = line
+                    break
+        
+        if todoType != None:
+            embedColor = int(todoType[1], 16)
+        else:
+            embedColor = 0x28a745
+        
+        newEmbed = discord.Embed(description=command[0], url="", color=embedColor)
+        
+        command[2] = command[2].replace(' ', '') #TODO: Use dfind instead ?
+        if command[2] != "false":
+            if command[2].startswith("<@"):
+                user = ctx.guild.get_member(int(command[2][2:-1]))
+            else:
+                user = ctx.guild.get_member_named(command[2])
+            
+            if user != None:
+                newEmbed.add_field(name="Asssigned to", value=user.mention, inline=True)
+        
+        if command[3] != "false":
+            if command[3].startswith("<#"):
+                repostChannel = ctx.guild.get_channel(int(command[3][2:-1]))
+            else:
+                for chan in ctx.guild.channels:
+                    if(chan.name == command[3]):
+                        repostChannel = chan
+        else:
+            repostChannel = None
+        
+        newEmbed.set_footer(text=command[1])
+        if repostChannel != None:
+            publicTodo = await repostChannel.send(embed=newEmbed)
+            newEmbed.add_field(name="Public repost", value=repostChannel.mention + " : " + str(publicTodo.id), inline=True)
```

Comment:
Again, camel case. And I'll stop writing this every time I see camel case again. I'll let you review you own code ;)